### ISSUE TITLE
[Bug]: InteractionTag does not rotate focus properly on tabbing

### ISSUE THREAD

**Issue by mscari-msft** [microsoft/fluentui#33250]
### Component

Tag

### Package version

9.49.2

### React version

16.14.0

### Environment

```shell
System:
    OS: Windows 11 10.0.22631
    CPU: (20) x64 Intel(R) Core(TM) i9-10900 CPU @ 2.80GHz
    Memory: 24.77 GB / 63.71 GB
  Browsers:
    Edge: Chromium (127.0.2651.98)
    Internet Explorer: 11.0.22621.3527
```

### Current Behavior

Using an InteractionTag and TagGroup does not allow a user to rotate focus between the tags. Coincidentally, using the arrow keys does work. Tab/Shift+Tab should work the same as the arrow keys so that users can navigate the tags using Tab.

### Expected Behavior

A user should be able to have focus on the first Tag, then hit tab to go to the next. This does not work.

### Reproduction

.

### Steps to reproduce

It can be reproduced on the Fluent documentation by trying to use Tab/Shift+Tab to change the focus in a group of Tags: https://react.fluentui.dev/?path=/docs/components-tag-interactiontag--default

### Are you reporting an Accessibility issue?

None

### Suggested severity

Urgent - No workaround and Products/sites are affected

### Are you willing to submit a PR to fix?

yes

### Validations

- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] The provided reproduction is a minimal reproducible example of the bug.

**Comment by bsunderhus** [comment]
`Tag`s in the `TagPicker` are grouped as a `listbox`  @mscari-msft , and this is the expected WAI ARIA navigation pattern for a `listbox`

**Comment by khmakoto** [comment]
Hi @mscari-msft, this is by design. If you really need to have tabbable `InteractionTags` you can add that behavior yourself really easily by using `useArrowNavigationGroup`. You can see this in action [in the following Stackblitz](https://stackblitz.com/edit/dtiuy4-yni6nc?file=src%2Fexample.tsx).

Let me know if there is anything else I can help you with.
